lib/apiForSheets.js

```diff
@@ -18,14 +18,17 @@ var clientSecret = process.env.CLIENT_SECRET;
 var context;
 
 /**
-  * Create an OAuth2 client with the given credentials, and then execute the
-  * given callback function.
-  *
-  * @param {Object} credentials The authorization client credentials.
-  * @param {function} callback The callback to call with the
-  *   authorized client.
-  */
-module.exports.authorize = function(credentials) {
+ * Create an OAuth2 client with the given credentials, and then execute the
+ * given callback function.
+ *
+ * @param {Object} credentials The authorization client credentials.
+ * @param {function} callback The callback to call with the
+ *   authorized client.
+ */
+
+//  This block is ignored because of the inability to mock googleAuth.
```

Comment:
Add the explanation what you provided me - i.e. why we are unable to mock googleAuth.

Reply:
Done.